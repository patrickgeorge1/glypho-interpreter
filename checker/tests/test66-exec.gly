===#YYY\YYYW;;;w///+zzzx`Fa`+:::w>>>