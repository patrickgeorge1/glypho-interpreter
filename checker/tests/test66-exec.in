345
67834334
0
235423
4326235
4326235
