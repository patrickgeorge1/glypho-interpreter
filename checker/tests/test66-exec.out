1
67834334
